lib/actions/auth.action.ts

```
@@ -0,0 +1,120 @@
+  const { email, idToken } = params;
+
+  try {
+    const userRecord = await auth.getUserByEmail(email);
+    if (!userRecord)
+      return {
+        success: false,
+        message: "User does not exist. Create an account.",
+      };
+
+    await setSessionCookie(idToken);
+  } catch (error: any) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)